config/routes.rb

```diff
@@ -65,7 +65,10 @@
     namespace :v2, path: '/v2' do
       resources :metacodes, only: [:index, :show]
       resources :mappings, only: [:index, :create, :show, :update, :destroy]
-      resources :maps, only: [:index, :create, :show, :update, :destroy]
+      resources :maps, only: [:index, :create, :show, :update, :destroy] do
+        post :stars, to: 'stars#create', on: :member
+        delete :stars, to: 'stars#destroy', on: :member
```

Comment:
If you use resources :stars, only: [:create, :destroy], then it'll make you use the star id in the URL for deletion, which is dumb